docs/design-language.md

```diff
@@ -225,6 +225,53 @@ Panels come with an assortment of configurable navigation controls:
 
 ### Examples
 
-<img src="/img/panels/panel-welcome.jpg" alt="Example of an app 'Welcome Panel'" title="App 'Welcome Panel'" width="435" height="513" class="img-example" />
-<img src="/img/panels/panel-feature.jpg" alt="Example of the 'Result/Feature Details' panel" title="'Result/Feature Details' panel" width="435" height="513" class="img-example" />
+<img src="/img/panels/panel-welcome.jpg" alt="Welcome Dialog Panel" title="Welcome Dialog Panel" width="415" height="493" class="img-example" /> <img src="/img/panels/panel-feature.jpg" alt="Features Panel" title="Features Panel" width="415" height="493" class="img-example" />
+
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface. Modal dialogs provide a flexible layout strategy to allow the user to put any content they want before the user and draws their focus to it, setting the rest of the app into a non-interactive state until it is dismissed. This allows important information to be displayed, and preserves the background information.
+
+### Examples
```

Comment:
Can we have the Modal examples show the whole application with the modal overlay and everything? We should also consider having a maximum of three examples.

Reply:
Can you please remove the browser UI from the top of the screen shots?

Reply:
The content area seems to be limited to images at 958px wide. Should we resize the images to match?

Reply:
Looks good